### ISSUE TITLE
Character Escaping for MAIL_PASSWORD

### ISSUE THREAD

**Issue by demonspork** [BookStackApp/BookStack#1935]
I have an email account with a space in the password. I have tried single quotes and double quotes around the password and the email authentication is still failing. If I put in the password without quotes bookstack fails to start with errors about unexpected whitespace.

What is the correct procedure?

MAIL_PASSWORD=no more spaces

**Comment by ssddanbrown** [comment]
Double quotes, around the value, should be enough to make this work. Are you sure it's the password that's the issue?

**Comment by demonspork** [comment]
Not so sure anymore. I switched entirely to a different mail server and was getting similar errors about Authentication failed, but now when I test right before replying to give you the exact error message I just get "554 5.5.1 Error: No Valid Recipients". 
My Setup:
Docker container on unraid
Mail Settings: server hostname, port 587, TLS, User/Password set, MAIL_FROM is also set but it doesn't have an impact.

I have tried sending the test email from 2 separate user accounts that go to do separate emails.

I have also manually sent messages with the same email configured in the bookstail mail settings and it can send via SMTP on the same port from other mail clients, to the same recipients I am testing with bookstack.
